src/oogeso/dto/oogeso_input_data_objects_multienergy.py

```diff
@@ -0,0 +1,246 @@
+from dataclasses import dataclass, is_dataclass, asdict, field
+from typing import List, Optional, Tuple, Any, Dict, Union
+import re
+from .oogeso_input_data_objects import *
+
+
+# Device types defined as part of "basic":
+# DevicePowerSourceData
+# DeviceSink_elData
+# DeviceStorage_elData
+
+
+@dataclass
+class DeviceSource_elData(DeviceData):
```

Comment:
Sould avoid "_" in class and dataclass naming. Do DeviceSourceElData and similar.

Reply:
OK, but this change requires changes also elsewhere, so I'll do that later